LiteCore/tests/LogEncoderTest.cc

```diff
@@ -274,6 +274,7 @@ TEST_CASE("Logging throw in c++", "[Log]") {
     string excMsg;
     const LogFileOptions prevOptions = LogDomain::currentLogFileOptions();
     try {
+        ExpectingExceptions x;
```

Comment:
Is this line necessary? I tried to make logging functions self-contained, not relying this line.